config_test/evg_settings.yml

```diff
@@ -33,6 +33,7 @@ repotracker:
     logfile: "/tmp/repotracker_test.log"
 
 api:
+    github_webhook_secret: "test"
```

Comment:
we will almost certianly need to update the configuration that's created as part of the settings override for testing in evergreen. that lives in a heredoc in self-tests.yml